oxd-server/src/main/java/org/gluu/oxd/server/guice/GuiceModule.java

```diff
@@ -34,5 +38,6 @@ protected void configure() {
         bind(DiscoveryService.class).in(Singleton.class);
         bind(ValidationService.class).in(Singleton.class);
         bind(StateService.class).in(Singleton.class);
+        bind(OpClientFactory.class).to(OpClientFactoryImpl.class).in(Singleton.class);
```

Comment:
This is correct, now we need same but in `TestGuiceModule` which bind to mock factory.
```
bind(OpClientFactory.class).to(MockOpClientFactory.class).in(Singleton.class);
```